example-dotnet/Program.cs

```diff
@@ -11,8 +22,134 @@ static void Main(string[] args)
             Say.hello("example");
             Console.WriteLine("Hello World!");
 
+	    // Usage of BCrypt.Net API

```

Comment:
Since we've commented everything so thoroughly, we can add a note here saying that this is meant to demonstrate vulnerable methods (the simplest case).

Reply:
This note is added.